charms/docker/docker.py

```diff
@@ -13,6 +13,8 @@ class Docker:
 
     def __init__(self, socket="unix:///var/run/docker.sock", workspace=None):
         '''
+        Create a Docker object with socket and optional workspace.
```

Comment:
pedantic :cop: - we aren't creating a socket, we are telling the wrapper which socket to funnel communication through.